test/functions/vass/subclass-to-ref-formal.chpl

```diff
@@ -0,0 +1,14 @@
+// Test that for a '[const] ref' formal, the type of the actual
+// must equal the type of the formal.
+// (Unless an implicit conversion comes to play.)
+
+class C    { }
+class D: C { }
+
+proc test(ref lhs: C) {}
```

Comment:
Is lhs supposed to be "const ref" here?  As it is written I think it should compile and execute without error.  Actually, even if it is const I don't see an obvious reason that passing a D shouldn't work.